internal/provider/publish_pipeline_resource.go

```diff
@@ -0,0 +1,113 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/mezmo/terraform-provider-mezmo/internal/client"
+	. "github.com/mezmo/terraform-provider-mezmo/internal/provider/models"
+)
+
+var (
+	_ resource.Resource              = &PublishPipelineResource{}
+	_ resource.ResourceWithConfigure = &PublishPipelineResource{}
+)
+
+func NewPublishPipelineResource() resource.Resource {
+	return &PublishPipelineResource{}
+}
+
+type PublishPipelineResource struct {
+	client client.Client
+}
+
+func (r *PublishPipelineResource) TypeName() string {
+	return PROVIDER_TYPE_NAME + "_publish_pipeline"
+}
+
+func (r *PublishPipelineResource) NodeType() string {
+	return "publish_pipeline"
+}
+
+func (r *PublishPipelineResource) TerraformSchema() schema.Schema {
+	return PublishPipelineResourceSchema()
+}
+
+func (r *PublishPipelineResource) NotConvertible() bool {
+	// FIXME: Import is not implemented yet. We need import/export to be able to handle child modules with this.
```

Comment:
Should we create a jira card and mention the ticket number in the comment?

Reply:
I've since made one and can note it here, sure.  LOG-20061